src/commands/appdistribution-distribute.ts

```diff
@@ -43,13 +50,42 @@ export const command = new Command("appdistribution:distribute <release-binary-f
     "--groups-file <file>",
     "path to file with a comma separated list of group aliases to distribute to",
   )
+  .option(
+    "--test-devices <string>",
+    "semicolon-separated list of devices to run automated tests on, in the format 'model=<model-id>,version=<os-version-id>,locale=<locale>,orientation=<orientation>'. Run 'gcloud firebase test android|ios models list' to see available devices. Note: This feature is in beta.",
+  )
+  .option(
+    "--test-devices-file <string>",
+    "path to file containing a list of semicolon- or newline-separated devices to run automated tests on, in the format 'model=<model-id>,version=<os-version-id>,locale=<locale>,orientation=<orientation>'. Run 'gcloud firebase test android|ios models list' to see available devices. Note: This feature is in beta.",
+  )
+  .option("--test-username <string>", "username for automatic login")
+  .option("--test-password <string>", "password for automatic login")
```

Comment:
Password is likely a sensitive field, right? It is not appropriate to supply it as a flag, since that will be persisted in bash history.